aspnetcore/blazor/call-javascript-from-dotnet.md

```diff
@@ -579,42 +582,46 @@ As a more detailed example, consider the following component that renders an int
 
 The corresponding JavaScript module, which should be placed at `wwwroot/mapComponent.js`, is as follows:
 
-```js
+```javascript
 import 'https://api.mapbox.com/mapbox-gl-js/v1.12.0/mapbox-gl.js';
 
-// TO MAKE THE MAP APPEAR YOU MUST ADD YOUR ACCESS TOKEN FROM https://account.mapbox.com
-mapboxgl.accessToken = '<your access token here>';
+mapboxgl.accessToken = '{ACCESS TOKEN}';
```

Comment:
I don't think you should remove the comment (`// TO MAKE THE MAP APPEAR...`). This is taken directly from https://docs.mapbox.com/mapbox-gl-js/api/.

The information about where to get an API token (https://account.mapbox.com) is essential for anyone who wants to make this code run correctly.

Reply:
It's below ... we're trying to get text out of code examples. See if the paragraph that follows covers it.

How about both? I'll add it back. I just want to make sure that we get a localized version of that guidance in the text.

I'll do it on the next commit.

Reply:
Ah OK, thanks. Both sounds good!

Reply:
Done! :+1: